l2-frontend/src/pages/ExtraNotification.vue

```diff
@@ -241,6 +251,14 @@ export default class ExtraNotification extends Vue {
     }
   }
 
+  savejson() {
+    const ids = this.toPrintNumbers;
+    window.open(`/forms/json-nofication?pk=[${ids}]`);
```

Comment:
Но ведь ids встанут как стринговый вариант массива или чего то ещё. 
Надо убрать квадратные скобки и вставить через json stringify

Reply:
http://127.0.0.1:8060/directions/pdf?napr_id=[227316,227315]

здесь также

Reply:
http://127.0.0.1:8060/forms/extra-nofication?pk=[227205]
аналогично

Reply:
Это аналогично такой строчке 
```suggestion
    window.open(`/forms/json-nofication?pk=[${ids.toString()}]`);
```

вроде бы это даёт результат, но ненадёжно.

Давай сделаем так
```suggestion
    window.open(`/forms/json-nofication?pk=${JSON.stringify(ids)}`);
```

Reply:
Я понял, что уже так сделано в других местах.
Но это криво, так не надо. Не знаю, как оно попало в репо, я это случайно сделал или на ревью не увидел. Но такого кода лучше не добавлять